components/blocks/cards/BasicAlert/BasicAlert.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+import BaseModal from 'elements/modals/BaseModal/BaseModal.vue'
+import CloseButton from 'elements/buttons/CloseButton/CloseButton.vue'
+import Card from 'elements/cards/Card/Card.vue'
+import CardHeader from 'elements/cards/CardHeader/CardHeader.vue'
+import CardBody from 'elements/cards/CardBody/CardBody.vue'
+import { useModalAlert } from 'composables/useModalAlert'
+
+const modalAlertState = useModalAlert()?.value
+
+const close = () => {
```

Comment:
can we rewrite this to `function close() => {`. I find using the `function` keyword to be a lot easier to instantly understand.